clients/vault/src/oracle/collector/proof_builder.rs

```diff
@@ -279,6 +283,7 @@ impl ScpMessageCollector {
 						}
 
 						let mut envelopes_map = envelopes_map_arc.write();
+						let mut from_arc_map = env_from_archive_map.write();
```

Comment:
Let's rename to `from_archive_map`. Otherwise one might confuse it with the `Arc<>` type of Rust

Reply:
done